New package: SampleDash v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: SampleDash
- Repository: https://github.com/nkottary/SampleDash.jl
- Created by: @nkottary
- Version: v0.1.0
- Commit: b0438c2a18b34ef8228f12b93fe4d2cbab0d1775
- Reviewed by: @nkottary
- Reference: https://github.com/nkottary/SampleDash.jl/commit/b0438c2a18b34ef8228f12b93fe4d2cbab0d1775#commitcomment-130487350
- Description: Example Dash application on JuliaHub
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6028e5143e81ab36c8c7f3518adbbd46074af9a6c31440a49783e1f72a16ac4cc31e006bf53921e1260633a898206e3c12f3b7b0db8b409b75cc2812d638def047078427ac5d9ed9ac91d8bc1a0e64a41e17bdd0257ec90bd5697c4d587e74ca037eb19c982e78536547c2ccc8c18110dc3952fa99bc20e31e988fb6a9a1adedf2d3ae0e13058c4a1ba8a1299dcdef4ac82bab817d26324f30e01d9ac00edaa24f81a7334f54684b4459474b33096239a -->

goerz commented:
Can you give some context for this, and extend the README and/or documentation? “Example” sounds like maybe this is not something that should be registered…

JuliaRegistrator committed:
New package: SampleDash v0.1.0

UUID: c21cdcaf-0d93-4d92-8cdb-99636d8a79d4
Repo: https://github.com/nkottary/SampleDash.jl.git
Tree: 2115940a0252503aaa96f5e1e21c7db285670e2d

Registrator tree SHA: f73a20c99934db92a256057d0d83ba394036a701
